src/bindings/js/node/include/preprocess/preprocess_wrap.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <napi.h>
+
+#include <openvino/openvino.hpp>
+
+#include "preprocess/resize_algorithm.hpp"
+#include "preprocess/pre_post_process_wrap.hpp"
+
+class PreProcessWrap {
```

Comment:
The name of this class is not descriptive. We don't wrap anything here.
We are trying to represent `ov::preprocess` namespace as js property that gathers preprocessing functionality